jazz_assets/Jenkinsfile_Platform

```diff
@@ -0,0 +1,51 @@
+#!groovy
+import groovy.json.JsonOutput
+import groovy.transform.Field
+
+// To be replaced as @Field def repo_credential_id = "value" except for config_loader
+@Field def repo_credential_id
+@Field def repo_base
+@Field def repo_core
+@Field def scm_type
+@Field def config_loader
+
+/*
+* Load environment variables from build module
+*/
+def loadConfigModule(build_module_url){
+  dir('config-loader') {
+    checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: repo_credential_id, url: build_module_url]]])
+    def result_json_string = readFile("jazz-installer-vars.json")
+    config_module = load "config-loader.groovy"
+    config_loader = config_module.initialize(result_json_string)
+  }
+}
+
+/*
+* Start the Pipeline stages
+*/
+node {
+  stage('Loading Config-Loader') {
+    loadConfigModule(getBuildModuleUrl())
+  }
+
+  stage('Triggering the Jenkins build') {
+    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: config_loader.JENKINS.CREDENTIAL_ID, passwordVariable: 'PWD', usernameVariable: 'UNAME']]){
+    echo "Build triggered via branch::${env.BRANCH_NAME}"
+
+    def build_job = config_loader.API.API_BUILD_URI
+    def job_url = JenkinsLocationConfiguration.get().getUrl() + build_job
+
+    sh "curl -X GET -k -v -u \"$UNAME:$PWD\"  \"" + job_url + "&service_name=assets&domain=jazz&scm_branch=${env.BRANCH_NAME}\""
```

Comment:
By the time this v1.7.1 will start building, we will have a new way of triggering builds: https://github.com/tmobile/jazz/pull/327/files#diff-09b7261183a16494507309dd77e3bc5eR34. Can you incorporate the changes ahead of time?

Reply:
Incorporated new changes as mentioned.